src/metatrain/experimental/gap/schema-hypers.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "enum": ["experimental.gap"]
```

Comment:
why `enum` and not string?

Reply:
I think that if you want to explicitly match a string to a specific value with jsonschema you have to use an enum